Formula/gphoto2.rb

```diff
@@ -22,4 +22,8 @@ def install
                           "--prefix=#{prefix}"
     system "make", "install"
   end
+
+  test do
+    `gphoto2 -v`
```

Comment:
Sorry for the crude test, I'll remember to use assertions next time.

I couldn't think of anything more extensive off the top of my head, since the cli pretty much depends on a connected camera, hopefully someone more experienced with the cli can think of something better.

Thanks for merging!